pennylane/vqe/vqe.py

```diff
@@ -175,6 +212,30 @@ def wires(self):
     def name(self):
         return "Hamiltonian"
 
+    @property
+    def grouping_indices(self):
+        """Return the grouping indices attribtue.
+
+        Returns:
+            list[list[int]]: indices needed to form groups of commuting observables
+        """
+        return self._grouping_indices
+
+    def get_groupings(self):
```

Comment:
I am not 100% sure about this - this is a hybrid of a function people can call to get the groupings, and to fill the `_grouping_indices` variable...

Reply:
Shall I add another method `compute_groupings` that just fills the indices? Or should it create a new Hamiltonian?

@Josh I am unsure about this - we want methods on operations to preferably return a new operation. However, this is such a minor alteration, it just stores some info.